### ISSUE TITLE
[BUG] Compile error with LAMMPS

### ISSUE THREAD

**Issue by stanmoore1** [lammps/lammps#1568]
**Summary**

Compile error: ../utils.cpp(179): error: identifier "atof" is undefined
    return atof(str);

**LAMMPS Version and Platform**

Latest LAMMPS master version, RHEL, intel compiler 16.0, openmpi-intel 1.10, gcc 4.9.3

**Expected Behavior**

LAMMPS compiles

**Actual Behavior**

See above.

**Steps to Reproduce**

Compile 

**Further Information, Files, and Links**

This fixes it for me:

```patch
$ git diff utils.cpp
diff --git a/src/utils.cpp b/src/utils.cpp
index bbda22f..8b58d2d 100644
--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -13,6 +13,7 @@
 
 #include <mpi.h>
 #include <cstring>
+#include <stdlib.h>
 #include "utils.h"
 #include "lammps.h"
 #include "error.h"
```

**Comment by akohlmey** [comment]
fixed in #1567. strange that this didn't show before. this is an obvious omission. hopefully the effort of #1542 will make this less likely, by reducing "include pollution" of header files.

@rbberger now that we have an updated intel license, can we have one compilation test with intel on pull requests? also, is the test on master active again?

**Comment by stanmoore1** [comment]
@akohlmey thanks
